Taarafo.Core/Brokers/Storages/StorageBroker.PostReports.References.cs

```diff
@@ -0,0 +1,28 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE TO CONNECT THE WORLD
+// ---------------------------------------------------------------
+
+using Microsoft.EntityFrameworkCore;
+using Taarafo.Core.Models.Posts;
+
+namespace Taarafo.Core.Brokers.Storages
+{
+    public partial class StorageBroker
+    {
+        private static void AddPostReportsReferences(ModelBuilder modelBuilder)
```

Comment:
This method has to be called on migrating.

Reply:
Ran this command: Add-Migration AddPostReportsReferences
Is this what you meant?